augur/refine.py

```diff
@@ -2,17 +2,17 @@
 Refine an initial tree using sequence metadata.
 """
 import numpy as np
-import os, shutil, time, sys
+import sys
 from Bio import Phylo
 from .dates import get_numerical_dates
 from .io import read_metadata
 from .utils import read_tree, write_json, InvalidTreeError
-from treetime.vcf_utils import read_vcf, write_vcf
+from treetime.vcf_utils import read_vcf
 from treetime.seq_utils import profile_maps
 
 def refine(tree=None, aln=None, ref=None, dates=None, branch_length_inference='auto',
              confidence=False, resolve_polytomies=True, max_iter=2, precision='auto',
-             infer_gtr=True, Tc=0.01, reroot=None, use_marginal=False, fixed_pi=None,
+             infer_gtr=True, Tc=0.01, reroot=None, use_marginal='always', fixed_pi=None, use_fft=True,
```

Comment:
Does this change the default argument for `use_marginal` when invoked via the CLI?

If so, we'd want to make this a major version increase for Augur.

Reply:
Silly question, it doesn't change CLI.

But FFT will now always used - and cannot be disabled via CLI.